modules/textures/src/lib/parsers/parse-crunch.d.ts

```diff
@@ -1,4 +1,4 @@
-import {TextureLevel} from '@loaders.gl/textures/types/texture';
+import {TextureLevel} from '../../types/texture.d.ts';
```

Comment:
I really do not want us to specify extensions in imports `.d.ts`. It should be enough to just do a relative import. And if we convert to `.ts` this should not be an issue.